### PR TITLE
Links order

### PR DESCRIPTION
**Pull request by franzmoro**
relates to #304

### CODE REVIEW

File: _harp/_partials/navlinks.ejs [hunk 1]
```diff
@@ -2,10 +2,10 @@
 	<a href="/"><h4 class="bold">HOME</h4></a>
 </li>
 <li>
-	<a href="/portfolio"><h4 class="bold">PORTFOLIO</h4></a>
+	<a href="/coders"><h4 class="bold">PROGRAMME</h4></a>
```
Comment (iteles) [inline _harp/_partials/navlinks.ejs]: Can we please change the route here too?

**Comment by iteles** [comment]
Totally agree with this 👍

**Comment by franzmoro** [comment]
@iteles not at all. tomorrow morning we'll do it.

**Comment by franzmoro** [comment]
Probably best to apply the re-ordering on the changes coming from #306, which already rename the directory, so that we don't counter @bradreeder 's work.

**Pull request closed without merging**